hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/checksum/BaseFileChecksumHelper.java

```diff
@@ -79,16 +82,16 @@ public abstract class BaseFileChecksumHelper {
     this.combineMode = checksumCombineMode;
     this.rpcClient = rpcClient;
     this.xceiverClientFactory =
-        ((RpcClient)rpcClient).getXceiverClientManager();
+        ((RpcClient) rpcClient).getXceiverClientManager();
     if (this.length > 0) {
       fetchBlocks();
     }
   }
 
   public BaseFileChecksumHelper(OzoneVolume volume, OzoneBucket bucket,
-      String keyName, long length,
-      OzoneClientConfig.ChecksumCombineMode checksumCombineMode,
-      ClientProtocol rpcClient, OmKeyInfo keyInfo) throws IOException {
+                                String keyName, long length,
+                                OzoneClientConfig.ChecksumCombineMode checksumCombineMode,
+                                ClientProtocol rpcClient, OmKeyInfo keyInfo) throws IOException {
```

Comment:
Could you undo these indentation changes?

Reply:
Thank you for the review. I have undone the indentation changes and also reverted some unnecessary changes caused by reformatting.